manifests/m/Microsoft/PIX/21.8.18.1/Microsoft.PIX.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.PIX
+PackageVersion: 21.8.18.1
+Installers:
+- Architecture: x86

```

Comment:
This is probably why its failing
```suggestion
- Architecture: x64
```
![image](https://user-images.githubusercontent.com/15158490/145656169-074ce6d6-fe2f-45e9-a64d-36e2d119718a.png)